### ISSUE TITLE
ability to not have main path be called _main

### ISSUE THREAD

**Issue by woldhek** [enzienaudio/heavy#60]
but define the main patch in the path of the patch (as an option).

Reason: i dont lik having all my patches called _main makes browsing the recent files in pd itself a PITA

**Comment by diplojocus** [comment]
Yeah we should expand the uploader script to accept a file path instead of just a directory.

**Comment by mhroth** [comment]
Would it be ok if you could give the uploader a file name, and it will assume that that is the `_main.pd`, also uploading all other Pd files in the directory?

**Comment by pingdynasty** [comment]
How about keeping the input directory argument as it currently is, with the option of declaring a main pd filename (which defaults to _main.pd). Then we've got backwards compatibility.

**Comment by woldhek** [comment]
both would work for me.
